content/applications/general/voip/transfer_forward.rst

```diff
@@ -0,0 +1,124 @@
+click the green :guilabel:`📞 (phone)` icon to start the call, and then click the red :guilabel:`📞
+(phone)` icon to end the call.
+
+.. image:: transfer_forward/manual-call.png
+   :align: center
+   :alt: Using the VoIP phone widget to make calls.
+
+Receive calls
+=============
+
+Receive calls into the Odoo :guilabel:`phone widget` easily. An incoming call opens up the
+:guilabel:`phone widget` when a user in using the Odoo database. Should the database be open in an
+open tab, then a sound plays (the sound must be activated on the device). Once back to the tab, the
+calling screen of the :guilabel:`phone widget` appears.
+
+Click the green :guilabel:`📞 (phone)` icon to pick up or the red :guilabel:`📞 (phone)` icon to
+reject the call.
+
+.. image:: transfer_forward/incoming-call.png
+   :align: center
+   :alt: Incoming call on the VoIP widget, with the call answer and call reject buttons highlighted.
+
+Add to call queue
+=================
+
+Keeping track of calls that need to be made on any given day can be cumbersome, but using the phone
+widget activities tracker calling is streamlined. All the contacts and customers that need to be
+made can be seen in one simple place on the Odoo phone widget, under the :guilabel:`Next activities`
+tab.
+
+To add a call to the :guilabel:`Next activities` tab simply click the green :guilabel:`📞 (phone)`
+icon in kanban view of the *CRM* application. To remove them again from the call queue, just hover
+over the opportunity with a call scheduled, and click on the red :guilabel:`📞 (phone)` icon that
+appears with the :guilabel:`- (minus)` icon. When navigating back to the *phone widget* only the
+calls that are scheduled immediately for that day appears in the queue under the :guilabel:`Next
+Activities` tab.
+
+.. image:: transfer_forward/add-call-queue.png
+   :align: center
+   :alt: Adding a call to the next activities tab in the VoIP phone widget.
+
+*Next activities* integration with the :guilabel:`phone widget` works with the following apps:
+*CRM*, *Project*, *Helpdesk*. A call can be added in the chatter of these records. To manually add a
+call via the chatter, click :guilabel:`Activities` (Next to the :guilabel:`🕗 (clock)` icon). Under
+:guilabel:`Activity Type`, select :guilabel:`Call` from the dropdown. Next, set a :guilabel:`Due
+Date` and add a :guilabel:`Summary`. Lastly, change the :guilabel:`Assigned to` field to the person
+that should make the call. Whomever is set in this last field (:guilabel:`Assigned to`), has this
+call show up in their :guilabel:`Next Activities` call queue in the :guilabel:`phone widget`.
+
+.. important::
+   Only calls for the immediate day (today's date) will appear in the :guilabel:`Next Activities`
+   tab of the :guilabel:`phone widget` for the specific user.
+
+If specified, click :guilabel:`Save` or :guilabel:`Open Calendar` to complete scheduling the call.
```

Comment:
```suggestion
All the contacts and customers that need to be called can be seen in one place with the Odoo *VoIP*
phone widget, under the :guilabel:`Next activities` tab.

To add a call to the :guilabel:`Next activities` tab, click the green :guilabel:`📞 (phone)` icon,
while in kanban view of the *CRM* application.

To remove them from the call queue, hover over the opportunity that has a call scheduled, and click
the red :guilabel:`📞 (phone)` icon that appears with the :guilabel:`- (minus)` icon.

When navigating back to the *VoIP* phone widget, **only** the calls that are scheduled immediately
for that day appear in the queue under the :guilabel:`Next Activities` tab of the *VoIP* pop-up
widget.

.. image:: transfer_forward/add-call-queue.png
   :align: center
   :alt: Adding a call to the next activities tab in the VoIP phone widget.

The :guilabel:`Next Activities` tab of the *VoIP* phone widget is integrated with the following Odoo
apps: *CRM*, *Project*, and *Helpdesk*.

A call can be added in the chatter of records within those applications.

To manually add a call, via the chatter, click :guilabel:`Activities` (next to the :guilabel:`🕗
(clock)` icon). Under :guilabel:`Activity Type`, select :guilabel:`Call` from the drop-down menu
that appears.

Next, set a :guilabel:`Due Date`, and add a :guilabel:`Summary`.

Lastly, change the :guilabel:`Assigned to` field to the person that should make the call. Whomever
is set in this last field (:guilabel:`Assigned to`), has this call show up in their :guilabel:`Next
Activities` call queue in the Odoo *VoIP* phone widget.

.. important::
   Only calls for the immediate day (today's date) appear in the :guilabel:`Next Activities` tab of
   the *VoIP* phone widget for that specific user.

If specified, click :guilabel:`Save`, or :guilabel:`Open Calendar`, to complete scheduling the call.
```
 I think a screenshot of the Next Activities tab in the VOIP pop-up widget could prove useful here.

Reply:
I've added in a screenshot. Made necessary edits.